manubot/process/manuscript.py

```diff
@@ -37,18 +37,18 @@ def get_text(directory):
     return '\n\n'.join(name_to_text.values()) + '\n'
 
 
-def replace_citations_strings_with_ids(text, string_to_id):
+def update_manuscript_citations(text, old_to_new):
```

Comment:
`old_to_new` is a generic name.  Would `old_to_short` be accurate and more descriptive?

Reply:
I did old_to_new because this function could be used to change citations to a new format other than short. In other words, function is more general, but our use case is specific.